csrc/scheduler/normalization_utils.cpp

```diff
@@ -1235,14 +1252,34 @@ void movePersistentBufferToSmem(
       // path of gmem -> smem to reduce temporary register usage. Otherwise, the
       // data path from gmem to shared memory (smem) follows this sequence: gmem
       // -> L1 cache -> register -> smem.
-      int hw_major = at::cuda::getCurrentDeviceProperties()->major;
-      if (is_cached_input && hw_major >= 8) {
+      if (supportCpAsync(tv) && is_cached_input) {
         tv->definition()->as<LoadStoreOp>()->setOpType(
             LoadStoreOpType::CpAsync);
         tv->definition()->as<LoadStoreOp>()->setCacheOp(CacheOp::Unspecified);
       }
+      // do a register cache for all the uses of this smem tv.
+      // The load from smem to register cache will then be vectorized to avoid
+      // bank conflicts. The determination of bank conflicts is made per
+      // transaction, with 16 bytes vectorized load, each warp needs 4
+      // transactions (32 threads * 16 bytes per thread / 128 bytes per
+      // transaction). In each transaction, different banks are visited, e.g.
+      // transaction-1, threads 0-7 visit banks 0-31
+      auto cached_tv = tv->cacheAfter();
```

Comment:
```suggestion
      auto cached_tv = tv->cacheAfter();
      // At this point, if cached_tv has multiple uses,  it becomes the persistent buffer instead of tv due to the way the persistent buffer selector works. To make tv remain as the persistent buffer, all of the uses must be privatized.
```